dicom-archive/dicomTar.pl

```diff
@@ -200,10 +200,10 @@ =head2 Methods
 
 
 # Remove all files starting with . in the dcm_source directory
-my $cmd = "cd " . $dcm_source . "; find -type f -name '.*' | xargs rm -f";
+my $cmd = "cd " . $dcm_source . "; find . -type f -name '.*' | xargs rm -f";
 system($cmd);
 # Remove __MACOSX directory
-my $cmd = "cd " . $dcm_source . "; find -name '__MACOSX' | xargs rm -rf";
+$cmd = "cd " . $dcm_source . "; find . -name '__MACOSX' | xargs rm -rf";
```

Comment:
I suggest doing this in one command:

```
my @args = ($dcm_source);
push(@args, qw/-type -f -name __MACOSX -o -name .* -delete/);
system('find', @args);
```

Note that I am using a form of `system` call that bypasses shell interpolation, so you don't have to quote `$dcm_source` (it will be interpreted *as is* by the command).